.env.template

```
@@ -3,4 +3,5 @@ Mongo_db = example_db
 PORT=5000
 clientId=GitHubClientId
 client_secret_m10=GitHubOAuthAppClientSecret
+LOGGLY_TOKEN
```

```suggestion
client_secret=GitHubOAuthAppClientSecret
LOGGLY_TOKEN=
LOG_LEVEL=DEBUG
```
Doesn't run without a log level